dictionaries/fr.json

```
@@ -468,6 +473,45 @@
       "viewProducts": "Voir les produits",
       "clientsSection": "Clients",
       "clientsDescription": "Utilisateurs assignés à cette entreprise",
+      "clientDetailsTitle": "Détails du client",
```

Biome (2.4.10)

[error] The key clientDetailsTitle was already declared.

(lint/suspicious/noDuplicateObjectKeys)